src/main/java/org/opentripplanner/standalone/config/OTPConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package org.opentripplanner.standalone.config;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+
+/**
+ * This class is responsible for loading OTP configuration. At the top level this is the
+ * 'otp-config.json', and at the graph level it is the 'build-config.json' and the
```

Comment:
We are re-introducing a third configuration file here (otp-config). We may really want only one or two config files, but will decide that when removing multiple router support.